ivadomed/keywords.py

```diff
@@ -194,6 +195,7 @@ class MetadataKW:
     INPUT_FILENAMES = "input_filenames"
     ROI_METADATA = "roi_metadata"
     PIXEL_SIZE = "PixelSize"
+    PIXEL_SIZE_UNITS = "PixelSizeUnits"
```

Comment:
Just a question, somewhat related to this PR, I see this keyword is added both at option/metadata KeyWords class. I need to review a bit more to tell but off the top of your head, do you think it is necessary to have these two separate classes? We did the keywords refactoring using best estimation from code perspective so not sure if we could have been more efficient or consolidated here?

Reply:
That is a good question!

So here is the catch:
- In the `MetadataKW` class, the name of the keywords come from the name of the JSON field as standardized in BIDS and indexed by pybids. So `PixelSize` and `PixelSizeUnits` in Pascal case, non-negotiable as per BIDS ;) They are used in the loader [here](https://github.com/ivadomed/ivadomed/blob/8090c86d63218a82b0c1f3c1709d36b6e1dcd47f/ivadomed/loader/segmentation_pair.py#L340).
- In the `OptionKW` class, the name of the keywords are based on the ivadomed variables name convention, so `pixel_size` and `pixel_size_units` in snake_case (I had them in PascalCase at the begining but was asked to changed them to snake case). In this case, the variable are unrelated to BIDS. They are used by the segment_volume API [here](https://github.com/ivadomed/ivadomed/blob/8090c86d63218a82b0c1f3c1709d36b6e1dcd47f/ivadomed/inference.py#L321).

A great example of their different usage is in the run_segment_command (when running `--segment`) [here](https://github.com/ivadomed/ivadomed/blob/8090c86d63218a82b0c1f3c1709d36b6e1dcd47f/ivadomed/main.py#L284), where the PixelSize (from BIDS metadata) is converted to pixel_sixe (for segment_volume options).

Reply:
Great! Thanks a lot for the detailed explanation.